src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -479,6 +479,7 @@ std::string ContextAttribute::render
       }
 
       out += valueTag(indent + "  ", ((ciP->outFormat == XML)? "contextValue" : "value"),
+                        "value",
```

Comment:
Not sure if the previous invocation was using this funtion incorrectly or if the addition of the "value" is due to the assocations removal. Could you  have a look?

Definetively, mixing polymorphism (specially when function signature is very close among them) with default values makes thinks very hard to follow :) (default values are ok, but in simple cases).